bandwidth.json

```diff
@@ -2431,10 +2431,10 @@
                 "type": "string",
                 "description": "The priority specified by the user.",
                 "enum": [
-                    "DEFAULT",
-                    "HIGH"
+                    "default",
```

Comment:
backward compatibility issue?

Reply:
I'm not sure what you mean, the API currently returns these in lowercase